client/src/components/siteContainer.js

```diff
@@ -4,10 +4,12 @@ import Auth from '../utils/auth';
 import NavBar from './navBar/index';
 import LandingPage from "../pages/landingPage/index";
 import Login from "../pages/login/index";
-import SignUpForm from '../pages/signUp/index';
+import SignUpForm from '../pages/signup/index';
```

Comment:
I don't think we want to change the signup form path. It should be camel-case "signUp"